src/app/chip-zcl-zpro/command-encoder/chip-zcl-zpro-codec.h

```diff
@@ -77,8 +77,12 @@ uint16_t encodeReadAttributesCommand(uint8_t * buffer, uint16_t buf_length, uint
 uint16_t encodeReadOnOffCommand(uint8_t * buffer, uint16_t buf_length, uint8_t destination_endpoint);
 
 /** @brief Extracts an aps frame from buffer into outApsFrame
+ * @param buffer Buffer to read from
+ * @param buf_length Length of buffer
+ * @param outApsFrame Pointer to EmberApsFrame struct to read into
+ * @return returns the number of bytes that were consumed to read out the EmberApsFrame. 0 means an error was encountered
  */
-bool extractApsFrame(uint8_t * buffer, uint32_t buf_length, EmberApsFrame * outApsFrame);
+uint16_t extractApsFrame(void * buffer, uint32_t buf_length, EmberApsFrame * outApsFrame);
```

Comment:
`buffer` arg should stay `uint8_t *`, no?

Reply:
Fixing in follow up PR